static/css/styles.css

```diff
@@ -0,0 +1,63 @@
+/* Profile page */
+
+.profile main {
+    display: block;
+}
+
+.profile h1 {
+    border: 0.3rem solid #00283C;
+    padding: 5rem;
+    text-align: center;
+    grid-area: title;
+}
+
+.profile main section {
+    padding: 1.5rem;
+}
+
+.profile main section h2{
+    text-align: center;
+}
+
+img.profile-pic {
```

Comment:
Why do you need a class for this?
Can't you do `.profile img` instead?

Reply:
if we have contributions written (right column on desktop), we'll have cover images. And I don't want them to behave the same way as profile pic. Should I use `.profile img:first-of-type` instead ?

Reply:
It just feels redundant to have profile > img > profile-pic.
Eventually the solution you suggest may feel more pertinent.

Reply:
ok I do that